Exiled.API/Features/Toys/Light.cs

```diff
@@ -94,6 +96,31 @@ public static Light Create(Vector3? position = null, Vector3? rotation = null, V
             return light;
         }
 
+        /// <summary>
+        /// Creates a new <see cref="Light"/>.
+        /// </summary>
+        /// <param name="position">The position of the <see cref="Light"/>.</param>
+        /// <param name="rotation">The rotation of the <see cref="Light"/>.</param>
+        /// <param name="scale">The scale of the <see cref="Light"/>.</param>
+        /// <param name="spawn">Whether the <see cref="Light"/> should be initially spawned.</param>
+        /// <param name="color">The color of the <see cref="Light"/>.</param>
+        /// <returns>The new <see cref="Light"/>.</returns>
+        public static Light Create(Vector3? position = null, Vector3? rotation = null, Vector3? scale = null, bool spawn = true, Color? color = null)
+        {
+            Light light = new(UnityEngine.Object.Instantiate(ToysHelper.LightBaseObject));
+
+            light.AdminToyBase.transform.position = position ?? Vector3.zero;
+            light.AdminToyBase.transform.eulerAngles = rotation ?? Vector3.zero;
+            light.AdminToyBase.transform.localScale = scale ?? Vector3.one;
+
+            if (spawn)
+                light.Spawn();
+
+            light.Color = color ?? Color.white;
```

Comment:
```suggestion
            light.Color = color ?? Color.gray;
```
Should match NW's default